runtime/trappist/src/xcm_config.rs

```diff
@@ -291,11 +314,20 @@ impl pallet_xcm::Config for Runtime {
 	type XcmTeleportFilter = Nothing;
 	type XcmReserveTransferFilter = Everything;
 	type Weigher = FixedWeightBounds<UnitWeightCost, RuntimeCall, MaxInstructions>;
-	type LocationInverter = LocationInverter<Ancestry>;
 	type RuntimeOrigin = RuntimeOrigin;
 	type RuntimeCall = RuntimeCall;
 	const VERSION_DISCOVERY_QUEUE_SIZE: u32 = 100;
 	type AdvertisedXcmVersion = pallet_xcm::CurrentXcmVersion;
+	type Currency = Balances;
+	type CurrencyMatcher = ();
+	type MaxLockers = ();
```

Comment:
it seems to better do it, otherwise `LockAsset` instruction won't work.

Reply:
right, but at the moment i'm not sure how we could use the LockAsset in this context, but as you mentioned it is better to set a value rather than leaving it empty and make it fail.

Reply:
probably not critical, but it's good that you noticed it now anyway.